lib/cmd.js

```diff
@@ -47,3 +31,22 @@ function isEmptyCommand (program) {
   return false;
 }
 exports.isEmptyCommand = isEmptyCommand;
+
+function validateProgram(program) {
+  Object.keys(program).filter(function (option) {
+    return ~DEBUG_VALUES.indexOf(option);
```

Comment:
So it seems as though this will always work but I'd rather this check if the result is `< 0`.

I know you didn't write it but I have a feeling other people will look at this and go "wtf?" then start trying it out and eventually realise that two's complement for bitwise operators is in the spec and so is `indexOf` returning `-1` so it *should* always work but really do we want to count on that? And why use `~` when you could just compare to `-1`? Oh well.